schemas/vax.schema

```diff
@@ -0,0 +1,35 @@
+SET client_min_messages TO WARNING;
+BEGIN;
+
+DROP SCHEMA IF EXISTS $PG_SCHEMA_VAX CASCADE;
+CREATE SCHEMA $PG_SCHEMA_VAX;
+
+CREATE TABLE $PG_SCHEMA_VAX.age (
+id   integer PRIMARY KEY,
+name varchar(28) NOT NULL
+);
```

Comment:
Adding a few extra columns would make this table more quantitative. For example: `age_0`, `age_1`, `is_high_risk`. Because the data source is unlikely to evolve, while not very elegant, 14 update queries would probably do the trick and do it fast. Something like this:

```
UPDATE vax.age
SET age0 = 0.5, age_1 = 17, is_high_risk = 0
WHERE name = '6 months - 17 years';
```

We could leave this for later though and do it if/when it becomes necessary.

Reply:
I agree and actually this is a fairly quick fix in the ingest code so will just make the update now.